python/kserve/kserve/protocol/dataplane.py

```diff
@@ -348,6 +350,31 @@ async def generate(
         response = await model.generate(request, headers=headers)
         return response, headers
 
+    async def completion(
+            self,
+            model_name: str,
+            request: Union[Dict, CompletionRequest],
```

Comment:
We can still use generate and replace `GenerateRequest` with `CompletionRequest`